Automattic/wp-calypso
# Earn: Add loaders to avoid content pop-in on Earn home screen

#### Changes proposed in this Pull Request

* This pull request updates the Earn home page blocks to display loading pulsers when we are still fetching membership/plan info.

#### Testing instructions



* Set account to premium or higher plan with Stripe connected. (PCYsg-lW4-p2, "Sandbox method" section)
* Navigate to `/earn/{site URL}`.
* Verify that the site displays two boxes with loading pulsers until all content is ready, rather than flickering between two different text states.
    * Slow down connection speed in browser dev tools if necessary.

### Before/After

![earn-pulser-before](https://user-images.githubusercontent.com/13437011/86638905-b2dfbf00-bf9d-11ea-846d-4bb401142615.gif)
![earn-pulser-after](https://user-images.githubusercontent.com/13437011/86956973-9b304400-c11f-11ea-8486-45519a823bf3.gif)

Fixes #43451
